### DISCUSSION TITLE
Forward HyperHDR Windows Screen Capture to HyperHDR running on RPI4

### DISCUSSION THREAD

**Discussion by pho084** [awawa-dev/HyperHDR#149] [Q&A] [answered]
Hi 

i have my LEDs connected to a RPI4 running HyperHDR
I want to use HyperHDR installed on Windows to capture the image and then forward the image to HyperHDR running on the RPI4.
Is that possible? 

I tried activating the forward option in Windows HyperHDR and pointing it to my RPI4's IP adress but that didn't work unfortunately

**Comment by awawa-dev** [comment, marked as answer]
Got some configuration issues because I've setup it long time ago but it works (v18beta Windows DX grabber => v18Beta Rpi4 aarch64, flatbuffers library has changed from v17 but according to their changelog there were no breaking changes)

SENDER UNIT (WINDOWS)
setup Rpi IP and target port
![obraz](https://user-images.githubusercontent.com/69086569/141506141-024f1129-398c-44ce-b0bc-b0673672c6bd.png)

RECEIVER UNIT (RPI)
make sure its the same receiver port set in the sender unit
![obraz](https://user-images.githubusercontent.com/69086569/141506342-e028b21e-ba10-41d8-9c07-6967fb2f0f32.png)
![obraz](https://user-images.githubusercontent.com/69086569/141506613-6a1223d2-58f4-4604-91a8-ca9ae0a4aa78.png)

If you can confirm it works on your setup too (with v17 or after upgrade to v18beta) please mark the discussion as answered.

**Reply by pho084** [reply]
First of all thanks for the reply and your awesome work/support 

...its weird though i did exactly that. I could also see the platform capture working via live preview on the windows client but the rpi4 'server' did not receive a signal. Both are v17. At the same time hyperionscreencapture works without a problem (but this is sending via proto buffer iiuc). I just wouldve loved to only use hyperHDR on both platforms.

Will try again when i am back home tonight. Thanks again in the meantime

**Reply by awawa-dev** [reply]
In my case at first I mistaken port number with protobuf service ...the problem is that it generates a lot of unrelated errors in the log. Just make sure the port is correct, both services should be enabled on both units and destination IP is reachable from the source unit (ping it first and put address rather as IP numbers not as a hostname). If won't work make sure that source unit has a video source (for example remotely accessed  DX grabber "likes" to disconnect when RDP session is minimized) from some grabber and validate both logs.

I think I tested flatbuffers somewhere in the middle of v17 developing so it should work also. Anyway you can find v18beta installers in the latest github action tab. It's stable enough at least for RPi platform as I almost always test beta versions on my production setup  ;)

**Comment by pho084** [comment]
oh jesus christ... turns out i am just stupid.... checked the logs, everything connected just fine, but still no image on the RPI4

i figured activating system capture in the settings would be enough but you actually have to manually activate it (again) in the REMOTE tab of the hyperHDR server... 

anyway it works just fine now, so thank you again. i will test this option against HyperionScreenCapture in terms of performance to try and answer my own thread from weeks ago. in games i saw some 10-20fps performance drop on the Windows machine with HSC, fingers crossed thats not the case with hyperHDR

**Reply by awawa-dev** [reply]
Great! Don't know what is performance penalty for the desktop but for the network consider to reduce the frame size that has to be sent because it seems HyperionScreenCapture  is using very aggressive setting to scale-down the captured image on default.
